src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+const PLACEHOLDER = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+const IMDB_URL = 'https://www.imdb.com/title/';
+
+interface Props {
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [hasError, setHasError] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const loadMovie = () => {
+    setIsLoading(true);
+
```

Comment:
If you have invalid title - don't send a request, also, put the error in such case